gulp/tasks/jsdoc.js

```diff
@@ -0,0 +1,44 @@
+var gulp    = require('gulp'),
+	jsdoc   = require('gulp-jsdoc'),
+	del 	= require('del');
+  
+
+gulp.task('jsdocs',function(){
+
+	// here we use a globbing pattern to match everything inside the 'documentation' folder
+	var folders = [
+    './documentation/**'
+    ];
+
+
+    var docStrapTemplate = {
```

Comment:
Isn't this a duplicate of the conf file in `gulp/util/jsdoc.conf.json`?

https://github.com/GoodBoyDigital/pixi.js/blob/dev/gulp/util/jsdoc.conf.json

Doc generation right now is through `npm run docs` which works fine. Do we need a gulp task as well?